modules/datastore.js

```diff
@@ -0,0 +1,39 @@
+// Imports the Google Cloud client library
+const {Datastore} = require('@google-cloud/datastore');
+
+const projectId = 'cloud-ad-step-2020';
+
+// Creates a client
+const datastore = new Datastore({
+  projectId: projectId,
+});
+
+/**This is the function for saving user's data
+    
+ */
+async function userData(name, email){
+    //The kind for the new entity being the user info
+    const kind = 'User';
+    //The Cloud Datastore key for the new entity
+    const userKey = datastore.key([kind, name]);
```

Comment:
Does this allow users with similar name?

Reply:
It should because there is a different id key for each user. My information only saves once because my email is the same but my name would save twice if the email was different because the token ID is different